testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -446,6 +446,18 @@ public void testMaterializedView()
         // view-specific listings
         checkInformationSchemaViewsForMaterializedView(schemaName, viewName);
 
+        // system.jdbc.tables without filter
+        assertThat(query("SELECT table_schem, table_name, table_type FROM system.jdbc.tables"))
+                .skippingTypesCheck()
+                .containsAll("VALUES ('" + schemaName + "', '" + viewName + "', 'TABLE')");
+
+        // system.jdbc.tables with table prefix filter
+        assertQuery(
+                "SELECT table_schem, table_name, table_type " +
+                        "FROM system.jdbc.tables " +
+                        "WHERE table_schem = '" + schemaName + "' AND table_name = '" + viewName + "'",
+                "VALUES ('" + schemaName + "', '" + viewName + "', 'TABLE')");
```

Comment:
This is "tables listing", so put it above of `// column listing`

also, let's have `system.jdbc.`  based "column listing" verification too

Reply:
moved the assertions
system.jdbc.columns needs some of the changes from https://github.com/trinodb/trino/pull/8113, currently it is empty just like information_schema.columns
i'll take care to add the tests for it along with the fixes for information_schema column listing